test/XRP/common-xrpl-client.test.ts

```diff
@@ -0,0 +1,228 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
this file is where we test the reliable submission logic by faking the networking.  pain in the ass